packages/ember-engines/tests/dummy/app/router.js

```diff
@@ -1,4 +1,4 @@
-import EmberRouter from '@ember/routing/router';
+import EmberRouter from '@embroider/router';
```

Comment:
@void-mAlex is this *required* to work with `ember-source` v5? Or, is it required only for Embroider scenarios?

Reply:
embroider, but I'm not separating the work, there is no point in doing so, having 2 prs doesn't change that this all needs to happen and it's a pain to pull it apart
happy to rename the pr to something more encompassing

Reply:
last time we split things apart ember v5 support snuck out early by way of peer dep relaxing wihtout actual support so if people tried it it would fail on them (unless it's a new project)